OPUNetGameSelectWnd.cpp

```diff
@@ -815,10 +823,12 @@ void OPUNetGameSelectWnd::OnClickJoin()
 	// Get the selected game to join
 	item.iItem = SendDlgItemMessage(this->hWnd, IDC_GamesList, LVM_GETSELECTIONMARK, 0, 0);
 	// Get the item data
-	if (SendDlgItemMessage(this->hWnd, IDC_GamesList, LVM_GETITEM, 0, (LPARAM)&item))
+	if (SendDlgItemMessage(this->hWnd, IDC_GamesList, LVM_GETITEM, 0, (LPARAM)&item)) {
 		joiningGame = (HostedGameInfo*)item.lParam;	// Retrieve the HostedGameListItem pointer
-	else
+	}
+	else {

```

Comment:
I often collapse these lines into:
```cpp
} else {
```

I find that helps associate the else block more strongly with the if block, rather than another independent block.
```cpp
if {
}
if {
}

if {
} else {
}
```

Reply:
I somewhat prefer not putting the else statement between 2 brackets on the same line. No big deal if you want to enforce that style though.

Reply:
Not a huge preference for me either, though `} else {` is the style I've been using since I switched to putting the opening `{` at the end of the `if` lines.